src/quo/react_native.cljs

```diff
@@ -55,6 +55,14 @@
 
 (def keyboard (.-Keyboard ^js rn))
 
+(defn alert [alert-title alert-description btns]
```

Comment:
if we move to quo2, probably we should move react native as well, but honestly im not sure if it should be in quo at all , we need to discuss